spec/v3.spec.json

```diff
@@ -159,6 +172,11 @@
           "type": "string",
           "pattern": "[a-zA-Z][a-zA-Z0-9_]{0,255}"
         },
+        "source": {
```

Comment:
It might be nice if this key was something like `sourceId` or `sourceIdentifier` or something like that to make it abundantly clear that this isn't the actual source code but rather a pointer to it.